prometheus/prometheus.yml

```diff
@@ -5,6 +5,9 @@ scrape_configs:
   - job_name: 'cleezy'
     static_configs:
       - targets: ['cleezy-app:8000']
+  - job_name: 'sceta-server'
+    static_configs:
+      - targets: ['server:8001']
```

Comment:
we should rename the prod sceta server to something else, will do it another day